### ISSUE TITLE
FR: Options to change font size

### ISSUE THREAD

**Issue by chrisgrieser** [audapolis/audapolis#376]
Thanks for the great app!

For accessibility reasons, there should be an option to increase font size.

**Comment by pajowu** [comment]
#400 addresses this by adding menu items & shortcuts (<kbd>Ctrl</kbd>+<kbd>+</kbd> for zooming in, <kbd>Ctrl</kbd>+<kbd>-</kbd> for zooming out and <kbd>Ctrl</kbd>+<kbd>0</kbd> for resetting the zoom). This does not only apply to the font size, but zooms everything. Is this enough or would you want a seperate option to only zoom the font?

**Comment by chrisgrieser** [comment]
I think this would be enough. Thanks a lot!
